apps/admin-gui/src/app/shared/components/dialogs/edit-attribute-definition-dialog/edit-attribute-definition-dialog.component.html

```diff
@@ -0,0 +1,183 @@
+            <mat-slide-toggle class="ml-2" color="accent"
+                              matTooltip="{{'DIALOGS.EDIT_ATTRIBUTE_DEFINITION.TOOLTIP_UNIQUE'| translate}}"
+                              matTooltipPosition="above"
+                              [matTooltipDisabled]="!disableUniqueToggle()"
+                              [disabled]="disableUniqueToggle()"
+                              [(ngModel)]="attDef.unique">
+              ON
+            </mat-slide-toggle>
+          </div>
+
+          <!--      <div class="d-flex flex-row justify-content-between mt-2">-->
+          <!--        <p class="ml-5 pl-5">{{'DIALOGS.EDIT_ATTRIBUTE_DEFINITION.SELF' | translate}}</p>-->
+          <!--        <p class="">{{'DIALOGS.EDIT_ATTRIBUTE_DEFINITION.SELF_PUBLIC' | translate}}</p>-->
+          <!--        <p class="">{{'DIALOGS.EDIT_ATTRIBUTE_DEFINITION.SELF_VO' | translate}}</p>-->
+          <!--        <p class="">{{'DIALOGS.EDIT_ATTRIBUTE_DEFINITION.VO' | translate}}</p>-->
+          <!--        <p class="">{{'DIALOGS.EDIT_ATTRIBUTE_DEFINITION.GROUP' | translate}}</p>-->
+          <!--        <p class="">{{'DIALOGS.EDIT_ATTRIBUTE_DEFINITION.FACILITY' | translate}}</p>-->
+          <!--      </div>-->
+
+          <!--      <div class="d-flex flex-row justify-content-between">-->
+          <!--        <p>{{'DIALOGS.EDIT_ATTRIBUTE_DEFINITION.READ' | translate}}</p>-->
+          <!--        <mat-checkbox [(ngModel)]="readSelf">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="readSelfPublic">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="readSelfVo">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="readVo">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="readGroup">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="readFacility">-->
+          <!--        </mat-checkbox>-->
+          <!--      </div>-->
+
+          <!--      <div class="d-flex flex-row justify-content-between">-->
+          <!--        <p>{{'DIALOGS.EDIT_ATTRIBUTE_DEFINITION.WRITE' | translate}}</p>-->
+          <!--        <mat-checkbox [(ngModel)]="writeSelf">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeSelfPublic">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeSelfVo">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeVo">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeGroup">-->
+          <!--        </mat-checkbox>-->
+          <!--        <mat-checkbox [(ngModel)]="writeFacility">-->
+          <!--        </mat-checkbox>-->
+          <!--      </div>-->
```

Comment:
The same here. If there is any reason, add it to it as a comment as well.

Reply:
Removed